content/applications/marketing/email_marketing/mailing_lists.rst

```diff
@@ -9,128 +9,212 @@ feedback, and more.
+   or select :guilabel:`Create and edit...` to create and edit the new mailing list right away.
+
+To remove a contact from a mailing list that the contact has been added to, enable the
+:guilabel:`Opt Out` checkbox. If the :guilabel:`Opt Out` checkbox is ticked, the ability to add a
+:guilabel:`Reason` why the contact opted-out is also available.
+
+When/if a contact *has* opted-out of a mailing list, the date they activated their opt-out appears
+in the :guilabel:`Unsubscription Date` column on their contact form.
+
+Lastly, the initial :guilabel:`Subscription Date` can be seen, as well. This field is auto-populated
+with the date and time their subscription to the mailing list has been saved.
+
+Multiple mailing lists can be added to a single contact form.
+
+To delete any mailing list from a contact form, simply click the :guilabel:`🗑️ (trash can)` icon.
+
+Link mailing list to website
+============================
 
 When a mailing list is created in the database, Odoo provides the option to directly link the
-mailing list on the Odoo-built website (created via Odoo's :guilabel:`Website` application).
+mailing list to the Odoo-built website (created via the Odoo *Website* application).
 
-To link a mailing list to a website, navigate to the front-end of the website, and enter
-:guilabel:`Edit` mode by clicking :guilabel:`Edit` in the upper-right corner. When clicked, Odoo
-reveals a right-sidebar, filled with drag-and-drop *Building Blocks* that are packed with various
-features, options, and design elements.
+To link a mailing list to a website, navigate to the front-end of the website, which can be
+accomplished in a variety of ways throughout the database. The most direct way to get to the
+front-end of the website is to simply open the :menuselection:`Website` application from the main
+Odoo dashboard.
 
-Then, to add a *subscription field* for a specific mailing list onto a website, drag-and-drop any of
-the :guilabel:`Newsletter` options (:guilabel:`Newsletter Block`, :guilabel:`Newsletter Popup`, or
-:guilabel:`Newsletter`).
+Doing so reveals the designated homepage of the Odoo-built website for the database.
 
-.. tip::
-   To quickly locate the :guilabel:`Newsletter` building block options (while in :guilabel:`Edit`
-   mode on the front-end of the website), type `Newsletter` into the search bar, located in the
-   right sidebar, under the :guilabel:`Blocks` tab, and Odoo reveals the three different
-   :guilabel:`Newsletter` block options.
+From the front-end of the website, click the :guilabel:`Edit` button in the upper-right corner. When
+clicked, Odoo reveals a right-sidebar, filled with drag-and-drop *building blocks*, packed with
+various features, options, and design elements.
+
+Next, in the search bar of the right-sidebar, search for `Newsletter`. The :guilabel:`Newsletter`
+selection of *building blocks* is used to add subscription fields for any mailing list onto the

```

Comment:
Since the term _building blocks_ was introduced in the last paragraph, I dont think it is necessary to keep using italics on it throughout the rest of the doc, LMK your thoughts!
```suggestion
selection of building blocks is used to add subscription fields for any mailing list onto the

```